plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -151,6 +159,8 @@ public MongoSession(TypeManager typeManager, MongoClient client, MongoClientConf
         this.client = requireNonNull(client, "client is null");
         this.schemaCollection = requireNonNull(config.getSchemaCollection(), "config.getSchemaCollection() is null");
         this.caseInsensitiveNameMatching = config.isCaseInsensitiveNameMatching();
+        this.authorizedDatabases = config.isMetadataPrivilegesRequired() == false;
```

Comment:
`!config.isMetadataPrivilegesRequired()`. Same below

Reply:
I removed property so this does not exists anymore